CMakeLists.txt

```diff
@@ -357,6 +357,17 @@ if(alignment_remainder)
 endif()
 message(STATUS "QMC_SIMD_ALIGNMENT is set to ${QMC_SIMD_ALIGNMENT}")
 
+#---------------------------------------------------------
+# Check for OpenMP compiler support if requested
+#---------------------------------------------------------
+if(QMC_OMP)
+  find_package(OpenMP)
+  if(NOT OpenMP_FOUND)
+      message(FATAL_ERROR "No compiler support for OpenMP..."
+                          "need to use -DQMC_OMP=OFF")
```

Comment:
Since OpenMP is a critical part of QMCPACK simulations I wonder if it's best to suggest finding compiler support for it. @prckent and @ye-luo thoughts?

Reply:
Will push an edit